components/list/list.html

```diff
@@ -0,0 +1,183 @@
+                    </li>
+                    <li class="item">
+                        <a href="../grid/grid.html">Grid</a>
+                    </li>
+                    <li class="item">
+                        <a href="../slider/slider.html">Slider</a>
+                    </li>
+                </ul>
+            </nav>
+            <div class="main-component">
+                <h1 class="comp-heading">List</h1>
+                <hr>
+                <p class="comp-desc">
+                    A list can be used to show multiple related options or links. Lists are less time taking to study as
+                    compared to paragraph.
+                    <br>
+                    <br>
+                </p>
+                <h2 class="comp-headng underline">Ordered List</h2>
+                <div class="comp-flex">
+                    <div class="comp-display">
+                        <ol class="numeric-ol">
+                            <li class="list-item">Item 1</li>
+                            <li class="list-item">Item 2</li>
+                            <li class="list-item">Item 3</li>
+                            <li class="list-item">Item 4</li>
+                        </ol>
+                        <ol class="alpha-ol">
+                            <li class="list-item">Item 1</li>
+                            <li class="list-item">Item 2</li>
+                            <li class="list-item">Item 3</li>
+                            <li class="list-item">Item 4</li>
+                        </ol>
+                    </div>
+                </div>
+                <div class="comp-code-display">
+                    <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C0.95%29&t=vscode&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Col%2520class%253D%2522numeric-ol%2522%253E%250A%253Cli%2520class%253D%2522list-item%2522%253EItem%25201%253C%252Fli%253E%253Cli%2520class%253D%2522list-item%2522%253EItem%25202%253C%252Fli%253E%253Cli%2520class%253D%2522list-item%2522%253EItem%25203%253C%252Fli%253E%253Cli%2520class%253D%2522list-item%2522%253EItem%25204%253C%252Fli%253E%250A%253C%252Fol%253E%250A%253Col%2520class%253D%2522alpha-ol%2522%253E%250A%253Cli%2520class%253D%2522list-item%2522%253EItem%25201%253C%252Fli%253E%253Cli%2520class%253D%2522list-item%2522%253EItem%25202%253C%252Fli%253E%253Cli%2520class%253D%2522list-item%2522%253EItem%25203%253C%252Fli%253E%253Cli%2520class%253D%2522list-item%2522%253EItem%25204%253C%252Fli%253E%250A%253C%252Fol%253E"
+  style="width: 100%; height: 20rem; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
+
+                </div>
+                <hr class="divider">
+                <div class="gutter"></div>
+                <h2 class="comp-headng underline">Unordered List</h2>
```

Comment:
Check all the typos in your code.

Reply:
Suggestion:- Use beautify or prettier to refactor your code. (not mandatory)

Else everything looks good.